run_tests.sh

```diff
@@ -627,12 +649,26 @@ if [ -n "$structured_data_report_file" ]; then
 else
     structured_data_args=""
 fi
-if [ -n "$with_framework_test_tools_arg" ]; then
-    GALAXY_TEST_TOOL_CONF="config/tool_conf.xml.sample,test/functional/tools/samples_tool_conf.xml"
+if [ "$with_framework_test_tools_arg" ]; then
```

Comment:
I think you can simplify the code by getting rid of `with_framework_test_tools_arg` and just set `GALAXY_TEST_TOOL_CONF` directly in the `case` above.